custom-set.json

```diff
@@ -1,90 +1,54 @@
-      "often by assigning variables in a sequence of operations.",
-      "This also is not represented here."
+     "These tests cover the core components of a set data structure: checking",
+     "presence, adding, comparing and basic set operations. Other features",
+     "such as deleting elements, checking size, sorting are not tested, but",
+     "you can add them if they are interesting in your language",
+     "",
+     "Tests about mixed-type sets are not included because the ability",
+     "to implement that will vary by language. If your language supports it",
+     "and you want to implement mixed-type sets, feel free."
    ],
-   "equal": {
-      "description": ["Test two sets for equality."],
+   "empty": {
+     "description": "Returns true if the set contains no elements",
+     "cases": [
+       {
+          "description": "sets with no elements are empty",
+          "set": [],
+          "expected": true
+       },
+       {
+          "description": "sets with elements are not empty",
+          "set": [1],
+          "expected": false
+       }
+     ]
+   },
+   "contains": {
+      "description": "Sets can report if they contain an element",
       "cases": [
          {
-            "description": "order doesn't matter",
-            "set1": [1, 3],
-            "set2": [3, 1],
-            "expected": true
-         },
-         {
-            "description": "dupicates don't matter",
-            "set1": [1, 1],
-            "set2": [1],
-            "expected": true
-         },
-         {
-            "description": "empty sets are equal",
-            "set1": [],
-            "set2": [],
-            "expected": true
-         },
-         {
-            "description": "set with single element is equal to itself",
-            "set1": [1],
-            "set2": [1],
-            "expected": true
-         },
-         {
-            "description": "different sets are not equal",
-            "set1": [1, 2, 3],
-            "set2": [3, 4, 5],
-            "expected": false
-         },
-         {
-            "description": "empty set is not equal to non-empty set",
-            "set1": [],
-            "set2": [1, 2, 3],
+            "description": "nothing is contained in an empty set",
+            "set": [],
+            "element": 1,
             "expected": false
          },
          {
-            "description": "non-empty set is not equal to empty set",
-            "set1": [1, 2, 3],
-            "set2": [],
-            "expected": false
+            "description": "when the element is in the set",
+            "set": [1, 2, 3],
+            "element": 1,
+            "expected": true
          },
          {
-            "description": "having most in common is not good enough",
-            "set1": [1, 2, 3, 4],
-            "set2": [2, 3, 4, 5],
+            "description": "when the element is not in the set",
+            "set": [1, 2, 3, 2],
```

Comment:
Does the duplicate `2` here serve a purpose? If so, can we fulfill that purpose without a duplicate?